### ISSUE TITLE
Stalker and PostgreSQL

### ISSUE THREAD

**Issue by kzweb** [azhurb/stalker_portal#256]
I wonder what the essential points that depend on MySQL, I intend to make a fork stalker and add support for PostgreSQL.

We are starting a new system in the company and would like to use all third parties in PostgreSQL.

Note: I've already contacted the service desk, I was told to open an issue at github.

Thank you.

**Comment by azhurb** [comment]
Unfortunately since version 5.0 DB module is located in the core, ie it is closed. Therefore, it would be difficult to do what you want.

**Comment by kzweb** [comment]
Yes. Searching for Github repository appears 183 results for "mysql". Perhaps changing the names by PhpStorm

Mysql::getInstance() -> DB::getInstance()
mysqlInstance -> dbInstance

Create an interface and from there to implement the methods. Soon after setting somewhere which driver to use.

It may be around the beginning.
